Readme.md

```diff
@@ -1,54 +1,54 @@
-
-![alt text](https://github.com/ShaileshSurya/go-images/blob/master/go_pic.jpg?raw=true)
-
 # Hammer [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![Build Status](https://travis-ci.org/ShaileshSurya/hammer.svg?branch=master)](https://travis-ci.org/ShaileshSurya/hammer) [![Coverage Status](https://coveralls.io/repos/github/ShaileshSurya/hammer/badge.svg?branch=master)](https://coveralls.io/github/ShaileshSurya/hammer?branch=master)
 
+Golang's Fluent HTTP Request Client
 
-Golang's Fluent HTTP Request Client 
-
-
+![alt text](https://github.com/ShaileshSurya/go-images/blob/master/go_pic.jpg?raw=true)
 
 ## Recipes
 
-```
+```go
 client := hammer.New()
 request, err := hammer.RequestBuilder().
-		<HttpVerb>().
-		WithURL("http://localhost:8081/employee").
-		WithContext(context.Background()).
-		WithHeaders("Accept", "application/json").
-		WithHeaders("user-id", "10062").
-		WithRequestParams("department", "HR").
-		Build()
+    <HttpVerb>().
+    WithURL("http://localhost:8081/employee").
+    WithContext(context.Background()).
+    WithHeaders("Accept", "application/json").
+    WithHeaders("user-id", "10062").
+    WithRequestParams("department", "HR").
+    Build()
 
 resp, err:= client.Execute(request)
 
-or 
+// or
 
-map := make(map[string]interface{})
-resp,err:= client.ExecuteInto(request, &map)
+m := make(map[string]interface{})
```

Comment:
Can we rename the `m` to map or more descriptive name like, responseMap?

Reply:
Fixed

Reply:
I think we missed this.

Reply:
Fixed for real this time